.golangci.yml

```diff
@@ -135,9 +135,10 @@ issues:
         - gocritic
         - dupl
         - goerr113
+        - nosnakecase
```

Comment:
we don't need to exclude rule here.

checking from your code, you are building entropy mock manually.
we can build it using library and ideally it should work without excluding rule.

steps:
1. build mock - ([example](https://github.com/goto/dex/blob/main/Makefile#L41))
2. remove the manually built mock in your test
3. use the generated mock - ([example](https://github.com/goto/dex/blob/main/internal/server/v1/alert/subscription_test.go#L38))

Reply:
this is nice!